src/theory/quantifiers/candidate_rewrite_filter.cpp

```diff
@@ -54,7 +55,7 @@ void CandidateRewriteFilter::initialize(SygusSampler* ss,
   ssn << "_dyn_rewriter_" << drewrite_counter;
   drewrite_counter++;
   d_drewrite = std::unique_ptr<DynamicRewriter>(
-      new DynamicRewriter(ssn.str(), &d_fakeContext));
+      new DynamicRewriter(d_env, &d_fakeContext, ssn.str()));
```

Comment:
This should also be changed to `d_drewrite = std::make_unique<DynamicRewriter>(d_env, &d_fakeContext, ssn.str());`